### PR TITLE
OCPBUGS-20076: Multus should determine kubeconfig path

### PR DESCRIPTION
**Pull request by dougbtv**
Otherwise, we have a report of a file not found due to a symlink issue

### CODE REVIEW

**Review by s1061123 — COMMENTED** [review]
/lgtm

**Comment by dougbtv** [comment]
/retitle OCPBUGS-20076: Multus should determine kubeconfig path

File: pkg/network/multus.go [hunk 1]
```diff
@@ -121,3 +124,21 @@ func pluginCNIConfDir(conf *operv1.NetworkSpec) string {
 	}
 	return MultusCNIConfDir
 }
+
+// determineKubeConfigPath determines which kubeconfig path to use for certificate generation.
+// In some cases, specifically IBM Managed Openshift, the path differs.
+func determineKubeConfigPath() string {
+	var kubeletKubeconfigPaths = []string{
+		"/var/lib/kubelet/kubeconfig",        // Traditional OCP
+		"/etc/kubernetes/kubelet-kubeconfig", // IBM Managed OpenShift, see OCPBUGS-19795
```
Comment (abhat) [inline pkg/network/multus.go:133]: nit: reference the new bug OCPBUGS-20064?

**Comment by abhat** [comment]
The metal ipi failure has nothing to do with this change. the job has been permafailing with an ironic error.

**Pull request merged**